DESCRIPTION

```diff
@@ -42,7 +42,7 @@ Encoding: UTF-8
 LazyData: true
 URL: https://github.com/rudeboybert/fivethirtyeight
 BugReports: https://github.com/rudeboybert/fivethirtyeight/issues
-RoxygenNote: 7.0.2
+RoxygenNote: 7.1.0
 Suggests:
     ggplot2,
     dplyr,
```

Comment:
@rudeboybert In `Suggests:` we should probably update `tidyr` to be `tidyr (>= 1.0.0)` to reflect the switch to `pivot_*`, right?

Reply:
good point. will fix